docs/configuration.asciidoc

```diff
@@ -46,44 +48,42 @@ public class Startup
 }
 ----
 
-NOTE: As explained in <<setup-asp-net-core>>, the `UseElasticApm` method only turns on ASP.NET Core monitoring. If you want to turn on everything (including HTTP and Database monitoring) you can use the `UseAllElasticApm` method from the `Elastic.Apm.NetCoreAll` package, which has the same overloads and behavior, except it turns on tracing for everything that is supported by the agent on .NET Core including database and HTTP tracing.
-
 With this you can use any configuration source that you configured on the `IConfiguration` instance that you passed to the APM Agent.
-You can find the key of each configuration option
-in the `IConfiguration or Web.config key` column of the corresponding option's description.
+You can find the key for each APM configuration option in this documentation, under the `IConfiguration or Web.config key` column of the option's description.
+
+NOTE: It is also possible to call `UseElasticApm()` without the overload. In this case, the agent will only read configurations from environment variables.
 
-NOTE: By simply calling `app.UseElasticApm()` without the overload, the agent will read configurations only from environment variables.
+NOTE: The `UseElasticApm` method only turns on ASP.NET Core monitoring. To turn on tracing for everything supported by the Agent on .NET Core, including HTTP and database monitoring, use the `UseAllElasticApm` method from the `Elastic.Apm NetCoreAll` package. Learn more in <<setup-asp-net-core,ASP.NET Core setup>>.
 
 [float]
 [[sample-config]]
 ==== Sample configuration file
 
-Below is a sample `appsettings.json` configuration file for a typical ASP.NET Core application. There are two important takeaways:
-
-1. The part below `ElasticApm` is fetched by the agent if the corresponding `IConfiguration` is passed to the agent.
-2. With ASP.NET Core, you must set `LogLevel` for the internal APM logger in the standard `Logging` section with the `ElasticApm` category name.
+Here is a sample `appsettings.json` configuration file for a typical ASP.NET Core application. There are two important takeaways, which are listed as callouts below the example:
 
 [source,js]
 ----
 {
   "Logging": {
-    "LogLevel": {
+    "LogLevel": { <1>
       "Default": "Warning",
       "Elastic.Apm": "Debug"
     }
   },
   "AllowedHosts": "*",
-  "ElasticApm":
+  "ElasticApm": <2>
     {
       "ServerUrls":  "http://myapmserver:8200",
       "SecretToken":  "apm-server-secret-token",
       "TransactionSampleRate": 1.0
     }
 }
 ----
+<1> With ASP.NET Core, you must set `LogLevel` for the internal APM logger in the standard `Logging` section with the `ElasticApm` category name.
```

Comment:
Should the "category name" be `Elastic.Apm` or `ElasticApm`?

Reply:
It's `Elastic.Apm`.

Reply:
Fixed